File: 프로그래머스/lv2/12941. 최솟값 만들기/최솟값 만들기.cpp
#include <bits/stdc++.h>
using namespace std;
int ret;
int solution(vector<int> A, vector<int> B)
{
    sort(A.begin(), A.end());
    sort(B.begin(), B.end(), greater<int>());
    
    for(int i = 0; i < A.size(); i++){
        ret += A[i] * B[i];
    }
    
    return ret;
}

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.*;
class Solution{
    public int solution(int []A, int []B){
        int ret = 0;
        int n = A.length;
        Arrays.sort(A);
        Arrays.sort(B);
        for(int i = 0; i < n; i++) {
            ret += A[i] * B[n - 1 - i];
        }        
        // System.out.println("Hello Java");
        return ret;
    }
}